### PR TITLE
EVG-13943: Link Legacy waterfall to project health 

### PR DESCRIPTION
**Pull request by SupaJoon**
[EVG-13943](https://jira.mongodb.org/browse/EVG-13943)

### Description 
Screen Shot 2022-04-14 at 4 03 01 PM

### CODE REVIEW

File: public/static/js/menu.js [hunk 1]
```diff
@@ -46,4 +46,5 @@ mciModule.controller("NavBarController", function (
 ) {
   $scope.newUILink = $window.NewUILink;
   $scope.showNewUILink = $window.user && $window.NewUILink;
+  console.log( $window.user, $window.NewUILink)
```
Comment (ablack12) [inline public/static/js/menu.js:49]: Is this debugging?

**Review by khelif96 — CHANGES_REQUESTED** [review]
Gonna Request Changes to block merging this PR. We shouldn't merge this until we are ready to launch

**Review by ablack12 — APPROVED** [review]
LGTM whenever we launch 🥳

**Review by khelif96 — APPROVED** [review]

**Comment by khelif96** [comment]
@SupaJoon  can you merge in main and make sure it still works since the pr has been stale?

**Pull request merged**